rasa/cli/scaffold.py

```diff
@@ -63,7 +63,6 @@ def print_train_or_instructions(args: argparse.Namespace, path: Text) -> None:
         print_success(
             "No problem 👍🏼. You can also train a model later by going "
             "to the project directory and running 'rasa train'."
-            "".format(path)
```

Comment:
I've simply removed this because of:
> F523 '...'.format(...) has unused arguments at position(s): 0

so basically, it was not shown to the user. Not sure we want to change this or not at this stage